library/OrderCloud.Catalyst/OrderCloud.Catalyst.csproj

```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
 	  <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-	  <Version>1.0.2</Version>
-
+	  <Version>1.0.4</Version>
```

Comment:
Did you accidentally skip a version?

Reply:
I accidentally skipped committing a version 😉